ENIGMAsystem/SHELL/Audio_Systems/SDL/Info/About.ey

```diff
@@ -0,0 +1,7 @@
+%e-yaml
+---
+
+Name: SDL
+Identifier: SDL
+Description: Audio playback support using SDL. A light compatibility layer is provided for GameMaker's classic sound API. Emphasis is placed on positional audio playback and audio processing features of more recent GameMaker versions.
+Author: Abhishek Kumar Jha
```

Comment:
This needs to include the depends and represents so the system can't be selected for the other platforms.
```yaml
Depends:
	Build-platforms: SDL

Represents:
	Build-platforms: SDL

```